example/cumulus-tf/main.tf

```
@@ -177,7 +181,7 @@ module "cumulus" {
 
   log_destination_arn = var.log_destination_arn
 
-  # Cumulus Distribution settings. Remove/comment if not using Cumulus Distribution.
+  # Cumulus Distribution settings. Remove/comment if not using Cumulus Distribution. (are using TEA)
```

What does `(are using TEA)` mean?

---

I means I couldn't parse the different between using cumulus distribution vs using TEA.
So not using Cumulus Distribution == using TEA.  Right above this we say "Thin Egress App settings. Uncomment to use TEA."
then here we say remove comment if not using Cumulus Distribution.  So now that I've explained it, what about? 
```suggestion
  # Cumulus Distribution settings. Remove/comment to use TEA```

---

```suggestion
  # Cumulus Distribution settings. Remove/comment to use TEA
```

You had some extra backticks in your suggestion. Otherwise, looks good